clients/admin-ui/src/features/system/SystemInformationForm.tsx

```diff
@@ -191,6 +216,15 @@ const SystemInformationForm = ({
     handleResult(result);
   };
 
+  const handleVendorSelected = (newVendorId: string) => {
+    if (features.tcf && newVendorId.split(".")[0] === "gvl") {
```

Comment:
let's use this helper func https://github.com/ethyca/fides/blob/main/clients/admin-ui/src/features/common/helpers.ts/#L168-L174